setup.py

```diff
@@ -1,21 +1,23 @@
+import sys
 import setuptools
 
-
 long_description = '''
-Texar is an open-source toolkit based on Tensorflow,
+Texar is an open-source toolkit based on TensorFlow,
 aiming to support a broad set of machine learning especially text generation
 tasks, such as machine translation, dialog, summarization, content manipulation,
 language modeling, and so on.
-
 Texar is designed for both researchers and practitioners for fast prototyping
 and experimentation. Checkout https://github.com/asyml/texar-pytorch for the
 PyTorch version which has the same functionalities and (mostly) the same
 interfaces.
 '''
 
+if sys.version_info < (3, 6):
+    sys.exit('Python>=3.6 is required by Texar.')
+
 setuptools.setup(
     name="texar",
-    version="0.2.4-unreleased",
+    version="0.2.5-unreleased",
```

Comment:
make it `0.4-unreleased` so that `0.3` can be reserved for TF-1.0  -- once GPT2 and decoders of TF-1.0 are refactored we'd release Texar TF 0.3

I'm also thinking of make tf2.0 as `import texar.tf2 as tx`